qemu/tests/uefishell.py

```diff
@@ -121,21 +123,28 @@ def send_command(self, command, check_result=False, interval=0.5):
 
         :param command: the command string being executed
         :param check_result: the pattern to validate output
+        :param interval: time interval between commands
+        :return if check_result is True, return matched string
```

Comment:
check_result is a pattern, i.e. a string, check_result='' or None instead of True or False.

Reply:
The last developer wrote it, I agree with you. I will update it.